tensorflow/docs_src/get_started/input_fn.md

```diff
@@ -368,6 +368,80 @@ Predictions: [ 33.30348587  17.04452896  22.56370163  34.74345398  14.55953979
   19.58005714]
 ```
 
+### Using Python reader
```

Comment:
Thank you for adding an example.
It would be better if this example only focuses on input_fn. to do that, please reuse an estimator mentioned above and only write an input_fn not a class.